tests/regression/package/test_package_regression.py

```diff
@@ -26,9 +26,7 @@ def tearDown(self):
             "aws-appsync-graphqlschema.yaml",
             "aws-appsync-resolver.yaml",
             "aws-appsync-functionconfiguration.yaml",
-            "aws-apigateway-restapi.yaml",
```

Comment:
Can you explain why these were removed?

It's ok to remove them but I want to ensure removing them does not loose coverage in testing unexpectedly.

Reply:
For this test, the hash generated inside SAM has a leading zero which is causing it to fail. I could maybe modify the value of the property a bit so we have a different hash (hopefully without leading zeros) and see if that works.

Reply:
Tried this and modified the SemanticVersion property. Now there is no value generated with leading zero. As far as I can tell, this changes does not affect the crux of the tests.